actors/miner/src/expiration_queue.rs

```diff
@@ -217,10 +249,12 @@ impl<'db, BS: Blockstore> ExpirationQueue<'db, BS> {
             .remove_active_sectors(sectors, sector_size)
             .map_err(|e| e.downcast_wrap("failed to remove sector expirations"))?;
 
+        // TODO it reschedules always as on time, add a flag?
```

Comment:
This one is for sector extension, isn't it? We probably need to separately reschedule those with a proof expiration earlier than new_expiration here. Probably similar to add_active_sectors – in fact may be simplest just to use that.

The method comment is now stale.

Reply:
Yeah, it was a comment for me, sorry I missed few of them.

Reply:
Ahh, this is the method I thought I removed, I think I ended up reverting that commit. It will be gone now.